README.md

```diff
@@ -132,7 +132,42 @@ Example: Train.jsonl
 
 ## Supported Models
 
-Current supported and tested models are `Llama3` (8B configuration has been tested) and `GPTBigCode`.
+Current supported and tested models are `Granite v3`, `Llama3` (8B configuration has been tested) and `GPTBigCode`.
+
+- For each tuning technique, we run testing on a single large model of each architecture type and claim support for the smaller models. For example, with QLoRA technique, we tested on granite-34b GPTBigCode and claim support for granite-20b-multilingual.
+
+- LoRA Layers supported : All the linear layers of a model + output `lm_head` layer. Users can specify layers as a list or use `all-linear` as a shortcut. Layers are specific to a model architecture and can be specified as noted [here](https://github.com/foundation-model-stack/fms-hf-tuning?tab=readme-ov-file#lora-tuning-example)
+
+- Legend:
+
+✅ Ready and available 
+
+✔️ Ready and available - compatible architecture (*see first bullet point above)
+
+🚫 Not supported
+
+?  may be supported, but not tested
```

Comment:
```suggestion
- Legend:

   ✅ Ready and available 

   ✔️ Ready and available - compatible architecture (*see first bullet point above)

   🚫 Not supported

   ?  may be supported, but not tested
```
Could indent these out a bit so they are inline with the text "Legend"

Reply:
I'm wondering if we need the ? since it's not being used in the table